packages/arb-token-bridge-ui/synpress.config.ts

```diff
@@ -239,7 +402,7 @@ function getWethContract(
 
 async function fundWeth(networkType: NetworkType) {
   console.log(`Funding WETH: ${networkType}...`)
-  const amount = networkType === 'parentChain' ? '0.2' : '0.1'
+  const amount = networkType === 'parentChain' ? '0.3' : '0.1'
```

Comment:
Changed to 0.3 because we are bridging WETH from parent chain now. If it stayed as it was we'd end up with 0.1 on both sides and cypress getters would throw